crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -884,6 +915,8 @@ async fn sys_validate_record_inner(
         )
         .await;
 
+        check_agent_validity(action, validation_dependencies.clone()).await?;
```

Comment:
When making a zome call, inline validation will always check agent validity, regardless of authority.